.eslintrc

```diff
@@ -135,7 +135,8 @@
           {
             "propName": "className",
             "message":
-              "Prop {{prop}} is forbidden. See https://github.com/Skyscanner/eslint-plugin-rules?tab=readme-ov-file#forbid-component-props"
+              "Prop {{prop}} is forbidden. See https://github.com/Skyscanner/eslint-plugin-rules?tab=readme-ov-file#forbid-component-props",
+            "allowedFor": ["InnerContainer"]
```

Comment:
We can still add a reason for each allowed value here like: 

```
[
"inner" // blah reason
]
```

Reply:
That's true! 😅